modules/broker-amqp/src/main/java/org/wso2/broker/amqp/codec/AmqpChannel.java

```diff
@@ -72,9 +91,9 @@ public ShortString consume(ShortString queueName, ShortString consumerTag, boole
                                ChannelHandlerContext ctx) throws BrokerException {
         String tag = consumerTag.toString();
         if (tag.isEmpty()) {
-            tag = UUID.randomUUID().toString();
+            tag = "sgen" + getNextConsumerTag();
```

Comment:
what is meant by `sgen`?

Reply:
shortcode for server-generated. Qpid has also used a similar thing